src/features/auth/pages/login/Login.native.test.tsx

```diff
@@ -358,9 +358,9 @@ describe('<Login/>', () => {
   })
 })
 
-const fillInputs = (renderAPI: ReturnType<typeof render>) => {
-  const emailInput = renderAPI.getByPlaceholderText('tonadresse@email.com')
-  const passwordInput = renderAPI.getByPlaceholderText('Ton mot de passe')
+const fillInputs = (screen: ReturnType<typeof render>) => {
```

Comment:
Justement, l'avantage de `screen`, c'est qu'il est accessible par tout le monde : plus besoin de le passer en paramètres ;) 
```suggestion
const fillInputs = () => {
```